frontend/src/Components/NavBar/NavBar.jsx

```diff
@@ -15,35 +15,64 @@ const liPadding = "px-3";
 
 /* MAIN */
 const NavBar = ({ loggedUser, logout }) => {
-  const history = useHistory();
-
-  const is = {};
-  if (loggedUser) {
-    if (loggedUser.admin) {
-      is["admin"] = true;
-    } else if (loggedUser.a_id) {
-      is["staff"] = true;
-    } else if (loggedUser.v_id) {
-      is["volunteer"] = true;
-    } else {
-      is["fellow"] = true;
-    }
+  // IS VARIABLE: determine user type and assign variable
+  // not passed down as prop from App because will use loggedUser to fetch userdata for avatar and name dnavModeplay on navbar
+  const navMode = {};
+  if (loggedUser && loggedUser.admin) {
+    navMode["admin"] = true;
+  } else if (loggedUser && loggedUser.a_id) {
+    navMode["staff"] = true;
+  } else if (loggedUser && loggedUser.v_id) {
+    navMode["volunteer"] = true;
   } else {
```

Comment:
Same thing here as for App.js, it's safer to don't default it to fellows

Reply:
added